cygnus-ngsi/conf/agent_ngsi.conf.template

```diff
@@ -197,7 +197,8 @@ cygnus-ngsi.sinks.postgresql-sink.type = com.telefonica.iot.cygnus.sinks.NGSIPos
 #cygnus-ngsi.sinks.postgresql-sink.batch_timeout = 30
 # number of retries upon persistence error
 #cygnus-ngsi.sinks.postgresql-sink.batch_ttl = 10
-
+# true if errors are persisted for this sink, false otherwise
+#cygnus-ngsi.sinks.postgresql-sink.persist_errors = true
```

Comment:
I think there are per-sink docu tables with the allowed parameters. Maybe persiste_errors should be added to all them.